### ISSUE TITLE
typo in productAccessibility schema template

### ISSUE THREAD

**Issue by FPa-riken** [HumanBrainProject/openMINDS_controlledTerms#16]
Missing initial capital in `_type`  value  => **P**roductAccessibility in [productAccessibility.schema.tpl.json](https://github.com/HumanBrainProject/openMINDS_controlledTerms/blob/v1/schemas/productAccessibility.schema.tpl.json)

**Comment by FPa-riken** [comment]
duplicate of https://github.com/HumanBrainProject/openMINDS_controlledTerms/pull/13
